packages/core/src/tools/create-lambda-handler.js

```diff
@@ -264,6 +264,13 @@ const createLambdaHandler = (appRawOrPath) => {
           const compiledApp = schemaTools.prepareApp(appRaw);
 
           const input = createInput(compiledApp, event, logger, logBuffer, rpc);
+
+          // When the app is done, callback() will be called and the Lambda function will end.
+          // In some cases, perhaps due to fetching large data blobs, the Lambda function
+          // will hang waiting for the event loop to drain. Setting this to false will
+          // prevent that and return when the callback is called.
+          context.callbackWaitsForEmptyEventLoop = false;
```

Comment:
🤔 I'm a bit nervous about enabling this by default. It seems like a good default for Lambda to wait until the event loop is empty before returning. Otherwise, I imagine we can introduce unpredictability that's quite hard to debug since some async tasks will complete sometimes and not other times.

We're actually explicitly setting this to `true` a bit above, [here](https://github.com/zapier/zapier-platform/blob/e8b49e027e98cef2b345e42d20707e520ab49f2f/packages/core/src/tools/create-lambda-handler.js#L192).

What about enabling this per-app to start with? At this point in the code we have access to the `event` payload from Lambda in which we could provide app-specific flags.